### PR TITLE
Add computed attribute

### PR DESCRIPTION
**Pull request by t-sommer**

### CODE REVIEW

File: schema/fmi3AttributeGroups.xsd [hunk 1]
```diff
@@ -102,14 +102,13 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
                 <xs:restriction base="xs:normalizedString">
                     <xs:enumeration value="constant"/>
                     <xs:enumeration value="fixed"/>
-                    <xs:enumeration value="calculated"/>
                     <xs:enumeration value="tunable"/>
-                    <xs:enumeration value="changing"/>
```
Comment (KarlWernersson) [inline schema/fmi3AttributeGroups.xsd:107]: Does the removal of changing originate from my PR?
It that case I belive it was a mistake

File: docs/2_4_common_schema.adoc [hunk 2]
```diff
@@ -704,17 +712,18 @@ This attribute must only be used for <<periodic-clock,periodic Clocks>>.
 latexmath:[\mathbf{T}_{interval} = ] `intervalCounter / resolution`.
 
 <<intervalCounter>> and <<resolution>> have no default value.
-This value must be greater than 0.
-This attribute is allowed if <<supportsFraction,`supportsFraction = true`>> and is required if <<intervalVariability,`intervalVariability = constant`>> or <<intervalVariability,`intervalVariability = fixed`>>.
-This attribute must only be used for <<periodic-clock,periodic Clocks>>.
+If present the value of this attribute must be greater than 0.
```
Comment (KarlWernersson) [inline docs/2_4_common_schema.adoc:715]: '0'

File: docs/2_4_common_schema.adoc [hunk 3]
```diff
@@ -678,14 +684,16 @@ The attribute `intervalVariability` declares the clock type, see <<table-overvie
...
+It must not be used otherwise.
 
 |`shiftDecimal`
 |
 [[shiftDecimal,`shiftDecimal`]]
 This attribute defines the delay of the first clock activation relative to latexmath:[\mathbf{t}_{\mathit{start}}] (<<startTime>>).
-This value must be equal or greater than 0.0.
-This attribute must only be used for <<periodic-clock,periodic Clocks>>.
+This attribute requires that <<intervalDecimal,`intervalDecimal`>> is present.
+The value of this attribute value must be equal to or greater than 0.0.
```
Comment (KarlWernersson) [inline docs/2_4_common_schema.adoc:695]: '0.0'

File: docs/2_4_common_schema.adoc [hunk 4]
```diff
@@ -678,14 +684,16 @@ The attribute `intervalVariability` declares the clock type, see <<table-overvie
...
+Default value is 0.0.
```
Comment (KarlWernersson) [inline docs/2_4_common_schema.adoc:696]: '0.0'

**Review by KarlWernersson — CHANGES_REQUESTED** [review]

File: schema/fmi3AttributeGroups.xsd (hunk 1 above)
Comment (andreas-junghanns) [reply schema/fmi3AttributeGroups.xsd:107]: I added that back into your initial PR - @t-sommer must rebase his proposal, I guess...

**Comment by andreas-junghanns** [comment]
@KarlWernersson : would you accept @t-sommer changes on top of yours?
Can we simplify all of this again and move this into one PR?

**Review by KarlWernersson — APPROVED** [review]
@andreas-junghanns 
Yes

**Pull request merged**